.circleci/config.yml

```diff
@@ -333,6 +336,13 @@ workflows:
             tags:
               only: /^v.*$/
 
+      - uat_tests:
+          requires:
+            - static_analysis
```

Comment:
What's the reason it requires only static_analysis ? Why having requirements at all?

Reply:
Good point, let me remove this.

My initial thought was ensuring swagger generated files were checked before UAT tests. Though no need for it.